src/script/main/app.js

```diff
@@ -716,7 +716,8 @@ class App {
       this.repository.calling.destroy();
 
       if (this.repository.user.isActivatedAccount()) {
-        if (isTemporaryClientAndNonPersistent(loadValue(StorageKey.AUTH.PERSIST))) {
+        const persist = loadValue(StorageKey.AUTH.PERSIST);
+        if (persist && isTemporaryClientAndNonPersistent(persist)) {
```

Comment:
That somehow doesn't make sense ... `persist && isTemporaryClientAndNonPersistent` 🤔

Reply:
Any suggestion? `persist` can be undefined that's why I have that `persist &&` check.

Reply:
What about `shouldPersist`?